src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx

```diff
@@ -61,18 +65,41 @@ export default function NftDrawer({
     savedCollectionSymbol,
   )
 
+  const defaultCollectionName = `${projectName} NFT rewards`
+  const defaultCollectionDescription = `The NFT rewards of the Juicebox project ${projectName}.`
+
   const onNftFormSaved = useCallback(async () => {
     setSubmitLoading(true)
-    // Calls cloud function to store NftRewards to IPFS
-    const CIDs = await uploadNftRewardsToIPFS(rewardTiers)
+    const rewardTiersCIDs = await uploadNftRewardsToIPFS(rewardTiers)
+    const nftCollectionMetadataCID = await uploadNftCollectionMetadataToIPFS({
+      collectionName: collectionName ?? defaultCollectionName,
+      collectionDescription: defaultCollectionDescription,
+      collectionLogoUri: logoUri,
+      collectionInfoUri: infoUri,
+    })
```

Comment:
this can be wrapped in a Promise.all([]) for a bit more speed.

Reply:
Done